deeppavlov/core/commands/train.py

```diff
@@ -154,14 +155,20 @@ def get_iterator_from_config(config: dict, data: dict):
     return iterator
 
 
-def train_evaluate_model_from_config(config: [str, Path, dict], iterator=None,
-                                     to_train=True, to_validate=True, download=False) -> Dict[str, Dict[str, float]]:
+def train_evaluate_model_from_config(config: [str, Path, dict], iterator=None, *,
+                                     to_train=True, to_validate=True, download=False,
+                                     recursive=True) -> Dict[str, Dict[str, float]]:
     """Make training and evaluation of the model described in corresponding configuration file."""
     config = parse_config(config)
 
     if download:
         deep_download(config)
 
+    if to_train and recursive:
+        for subconfig in get_all_elems_from_json(config['chainer'], 'config_path'):
```

Comment:
Currently, we can set 'config_path' in any inner attribute of a class in the config.  So I suggest searching 'config_path' recursively in the entire config, not only in first-level elements. Is it possible?
Example from my config (I have to do that to emulate a cycle inside my config):
```
{
        "name": "logit_ranker",
        "squad_model": {
          "config_path": "../deeppavlov/configs/squad/multi_squad_noans.json"
        },
        "top_n": 3,
        "sort_noans": true,
        "in": [
          "chunks",
          "questions"
        ],
        "out": [
          "best_answer",
          "context_indices"
        ]
      },
```

Reply:
Oh, I see it's already recursive in `get_all_elems_from_json()`. Alright then!